wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsStoreClient.java

```diff
@@ -40,8 +40,13 @@
 import org.hpccsystems.ws.client.wrappers.ArrayOfEspExceptionWrapper;
 
 /**
- * Use as soap client for HPCC WsFileIo web service.
- * This includes creating a new file, and appending data to a file in the given SHPCC System.
+ * Facilitates access to HPCC key/value based Storage.
```

Comment:
does it make sense to use HPCC Systems key/value based Storage. (and should storage be init lower case)